doc/common_pitfalls.rst

```diff
@@ -225,3 +225,346 @@ Below are some tips on avoiding data leakage:
   leakage as it ensures that the appropriate method is performed on the
   correct data subset. The pipeline is ideal for use in cross-validation
   and hyper-parameter tuning functions.
+
+.. _randomness:
+
+Controlling randomness
+======================
+
+Some scikit-learn objects are inherently random. These are usually estimators
+(e.g. :class:`~sklearn.ensemble.RandomForestClassifier`) and cross-validation
+splitters (e.g. :class:`~sklearn.model_selection.KFold`). The randomness of
+these objects is controlled via their `random_state` parameter, as described
+in the :term:`Glossary <random_state>`. This section expands on the glossary
+entry, and describes good practices and common pitfalls w.r.t. to this
+subtle parameter.
+
+.. note:: Recommendation summary
+
+    For an optimal robustness of cross-validation (CV) results, pass
+    `RandomState` instances when creating estimators, or leave `random_state`
+    to None. Passing integers to CV splitters is usually the safest option,
+    although `RandomState` instances are fine if you know what you are doing.
+    For both estimators and splitters, passing an integer vs passing an
+    instance (or None) leads to subtle but significant differences,
```

Comment:
change None to `None` ?

Reply:
@reshamas Just as info: You can even select one or several lines of code and then write as review comment:
\```suggestion
your version of the code
\```

Reply:
thanks @lorentzenchr !  This is helpful.